source/replication/server-side-replication-resynchronize-remote.rst

```diff
@@ -0,0 +1,546 @@
+
+- Replace :mc-cmd:`ALIAS <mc version ALIAS>` with the
+  :mc:`alias <mc alias>` of the MinIO cluster.
+
+- Replace :mc-cmd:`PATH <mc version ALIAS>` with the bucket on which
+  to enable versioning.
+
+.. _minio-bucket-replication-serverside-resynchronize-permissions:
+
+Required Permissions
+~~~~~~~~~~~~~~~~~~~~
+
+Bucket replication requires specific permissions on the source and
+destination deployments to configure and enable replication rules. 
+
+.. tab-set::
+
+   .. tab-item:: Replication Admin
+
+      The following policy provides permissions for configuring and enabling
+      replication on a cluster. 
+
+      .. literalinclude:: /extra/examples/ReplicationAdminPolicy.json
+         :class: copyable
+         :language: json
+
+      - The ``"EnableRemoteBucketConfiguration"`` statement grants permission
+        for creating a remote target for supporting replication.
+
+      - The ``"EnableReplicationRuleConfiguration"`` statement grants permission
+        for creating replication rules on a bucket. The ``"arn:aws:s3:::*``
+        resource applies the replication permissions to *any* bucket on the
+        source cluster. You can restrict the user policy to specific buckets
+        as-needed.
+
+      Use the :mc-cmd:`mc admin policy add` to add this policy to *both*
+      deployments. You can then create a user on both deployments using
+      :mc-cmd:`mc admin user add` and associate the policy to those users
+      with :mc-cmd:`mc admin policy set`.
+
+   .. tab-item:: Replication Remote User
+
+      The following policy provides permissions for enabling synchronization of
+      replicated data *into* the cluster. Use the :mc-cmd:`mc admin policy add`
+      to add this policy to *both* deployments.
+
+      .. literalinclude:: /extra/examples/ReplicationRemoteUserPolicy.json
+         :class: copyable
+         :language: json
+
+      - The ``"EnableReplicationOnBucket"`` statement grants permission for 
+        a remote target to retrieve bucket-level configuration for supporting
+        replication operations on *all* buckets in the MinIO cluster. To
+        restrict the policy to specific buckets, specify those buckets as an
+        element in the ``Resource`` array similar to
+        ``"arn:aws:s3:::bucketName"``.
+
+      - The ``"EnableReplicatingDataIntoBucket"`` statement grants permission
+        for a remote target to synchronize data into *any* bucket in the MinIO
+        cluster. To restrict the policy to specific buckets, specify those 
+        buckets as an element in the ``Resource`` array similar to 
+        ``"arn:aws:s3:::bucketName/*"``.
+
+      Use the :mc-cmd:`mc admin policy add` to add this policy to *both*
+      deployments. You can then create a user on both deployments using
+      :mc-cmd:`mc admin user add` and associate the policy to those users
+      with :mc-cmd:`mc admin policy set`.
+
+MinIO strongly recommends creating users specifically for supporting 
```

Comment:
is it user or service account?

Reply:
I suppose it could be either, but an SA is tied to a user. Deleting the user would delete the SA and kill replication. If the admin wasn't tracking what is being used where, it would result in an unexpected failure in replication.

I think we should recommend a full user dedicated to managing replication. This is easier to track via auditing, etc. as well.

Reply:
It should be service accounts @ravindk89 in case of LDAP setups `users` won't make sense.